specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/examples/PutPrivateLinkAssociation.json

```diff
@@ -0,0 +1,26 @@
+{
+  "parameters": {
+    "plaId": "00000000-0000-0000-0000-000000000000",
+    "groupId": "my-management-group",
+    "api-version": "2020-05-01",
+    "parameters": {
+      "privateLink": "00000000-0000-0000-0000-000000000000",
+      "publicNetworkAccess": "Enabled"
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "properties": {
```

Comment:
`properties`, `id`, `type`, and `name` needs to be removed to resolve model validation errors.